tests/test_picture.py

```diff
@@ -62,13 +62,62 @@ def test_Picture_MakePlaceholder():
         skia.Picture.MakePlaceholder(skia.Rect(100, 100)), skia.Picture)
 
 
-def test_PictureRecorder_init():
-    assert isinstance(skia.PictureRecorder(), skia.PictureRecorder)
+def test_RTreeFactory_init():
```

Comment:
Why have you removed the test_PictureRecorder_init (without adding a replacement, AFAIC)?

Reply:
Sorry, that's a mistake. Have re-added it.